pycbc/inference/sampler/emcee_pt.py

```diff
@@ -115,12 +117,29 @@ def from_config(cls, cp, model, nprocesses=1, use_mpi=False):
         # get the number of walkers to use
         nwalkers = int(cp.get(section, "nwalkers"))
         # get the number of temps
-        ntemps = int(cp.get(section, "ntemps"))
+        import h5py
+        if cp.has_option(section, "ntemps") and \
+                cp.has_option(section, "inverse_temperatures_input_file"):
+            raise ValueError("Must specify either ntemps or "
+                             "inverse-temperatures-input-file, not both.")
+        if cp.has_option(section, "inverse_temperatures_input_file"):
+            inverse_temperatures_input_file = cp.get(
+                                         section,
+                                         "inverse_temperatures_input_file")
```

Comment:
There's some inconsistency here; should it be `inverse_temperature_input_file` or `inverse-temperature-input-file`? Personally, I'd prefer the later. And why not just call it `betas-file`? That's easier to type.

Reply:
I had `betas-file` initially when I made this patch last time. But Duncan suggested this more verbose name as betas can be used for something other than inverse temperatures too.

Reply:
I've now made it `inverse-temperatures-input-file`.

Reply:
Oh ok... but how about dropping the 'input' (`inverse-temperatures-file`)?